### PR TITLE
Time bug fix

### PR DESCRIPTION
**Pull request by luciferlinx101**
### Description

- Time Bug Fix for backend for each feed 
- Added node_modules in .gitignore for project's root level

### Solution and Design

Fixed time bug issue by adding ```time_difference``` in ```/get/execution/{agent_execution_id}``` end point and in ```/get/agent/{agent_id}```

### Test Plan

- Time difference can be seen in API response 
- Added Unit test

### Type of change
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update

### Checklist
- [X] My pull request is atomic and focuses on a single change.
- [X] I have read the contributing guide and my code conforms to the guidelines.
- [X] I have documented my changes clearly and comprehensively.
- [X] I have added the required tests.

### CODE REVIEW

File: superagi/helper/time_helper.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    parsed_timestamp2 = datetime.strptime(timestamp2, time_format)
+
+    # Calculate the time difference
+    time_difference = parsed_timestamp2 - parsed_timestamp1
+
+    # Convert time difference to total seconds
+    total_seconds = int(time_difference.total_seconds())
+
+    # Calculate years, months, days, hours, and minutes
+    years, seconds_remainder = divmod(total_seconds, 31536000)  # 1 year = 365 days * 24 hours * 60 minutes * 60 seconds
```
Comment (TransformerOptimus) [inline superagi/helper/time_helper.py:18]: multiply here only.

File: superagi/models/toolkit.py [hunk 2]
```diff
@@ -5,9 +5,8 @@
 
 from superagi.models.base_model import DBBaseModel
 
-# marketplace_url = "https://app.superagi.com/api/"
-
 
+# marketplace_url = "https://app.superagi.com/api"
 marketplace_url = "http://localhost:8001"
```
Comment (TransformerOptimus) [inline superagi/models/toolkit.py:10]: remove this change

File: superagi/helper/feed_parser.py [hunk 3]
```diff
@@ -13,6 +16,9 @@ def parse_feed(feed):
               If parsing fails, the original feed is returned.
     """
 
+    # Get the current time
+    feed.time_difference = get_time_difference(feed.updated_at, str(datetime.now()))
```
Comment (TransformerOptimus) [inline superagi/helper/feed_parser.py:20]: add unit test for this.

Comment (luciferlinx101) [reply superagi/helper/feed_parser.py:20]: Added

File: superagi/helper/time_helper.py (hunk 1 above)
Comment (luciferlinx101) [reply superagi/helper/time_helper.py]: done!

**Review by TransformerOptimus — APPROVED** [review]

**Pull request merged**